src/fe-text/irssi.c

```diff
@@ -74,11 +75,20 @@ void mainwindow_activity_deinit(void);
 void mainwindows_layout_init(void);
 void mainwindows_layout_deinit(void);
 
+#ifdef HAVE_SECCOMP
```

Comment:
That would require far more ifdef statements, and they would be scattered all over sandbox.c, rather than put in one place in irssi.c. IMO, I think that putting the ifdef in irssi.c makes much more sense and you can instantly tell what its purpose is.